.eslintrc.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  'env': {
+    'browser': true,
+    'commonjs': true,
+    'es2021': true,
+    'mocha': true,
+    'node': true
+  },
+  'extends': 'eslint:recommended',
+  'overrides': [],
+  'parserOptions': {
+    'ecmaVersion': 'latest'
+  },
+  'rules': {
+    'brace-style': ['warn', 'allman'],
```

Comment:
wouldn't we prefer to support stroustrup?

Reply:
`stroustrop`? don't you mean `1tbs`? https://eslint.org/docs/latest/rules/brace-style#1tbs